src/odemis/gui/cont/tabs.py

```diff
@@ -632,13 +634,11 @@ def get_display_priority(cls, main_data):
     @call_in_wx_main
     def display_acquired_data(self, data):
         # get the top right view port
-        views = self.tab_data_model.views.value[1]
-        for s in views.getStreams():
-            views.removeStream(s)
-        streams = data_to_static_streams(data)
-        for s in streams:
-            views.addStream(s)
-
+        view = self.tab_data_model.views.value[1]
+        for s in data_to_static_streams(data):
+            stream_cont = StreamController(self._panel.pnl_cryosecom_acquired, s, self._tab_data_model,
+                                           show_panel=True, view=view, sb_ctrl=self._overview_stream_controller)
```

Comment:
What's the reason you're explicitly creating the StreamController, instead of calling the _overview_stream_controller.addStream() ? Is that the StaticStream issue that Bassim mentioned? Or is there more reason? Ideally, using the addStream() function would be better.

Reply:
I now see your previous answer explained it already.